doc/user_download_and_install_windows.rst.template

```diff
@@ -0,0 +1,355 @@
+    : MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    : GNU General Public License for more details.
+    :
+    : You should have received a copy of the GNU General Public License
+    : along with this program; if not, see <http://www.gnu.org/licenses/>
+    :
+    : --
+
+${big_fat_warning}
+
+Download and installation on Windows 10 64-bit (v. 1607)
+########################################################
+
+
+Disclaimer
+==========
+
+HORTON has been tested on Ubuntu 14.04 subsystem available on Windows 10 64-bit (after
+Windows 10 Anniversary Update i.e. v.1607). The Ubuntu 14.04 subsystem is not available to
+any Windows operating systems prior to the Windows 10 (v.1607) or 32-bit Windows system.s
+Cygwin based installation is not supported (although doable in theory).
+
+
+Enable Bash on Windows 10
+=========================
+Provided you are using the appropriate version of Windows 10 (64-bit, after Window 10
+Anniversary Update), go to "Settings > Update & Security > For Developers" (or
+alternatively search "Use Developer Features") and activate "Developer Mode". Then, go to
+"Programs and Features" and click "Turn Windows Features On or Off". Check the box
+"Windows Subsystem for Linux (Beta)". Restart Computer. Search "Bash" to open Ubuntu 14.04
+subsystem terminal.
+
+
+Download the code
+=================
+
+The latest stable source code release of HORTON can be downloaded here:
+
+    https://github.com/theochem/horton/releases/download/2.0.1/horton-2.0.1.tar.gz
+
+Choose a suitable directory, e.g. ``~/build``, download and unpack the archive
+
+.. code-block:: bash
+
+    mkdir -p ~/build
+    cd ~/build
+    curl -kLO https://github.com/theochem/horton/releases/download/2.0.1/horton-2.0.1.tar.gz
+    tar -xvzf horton-2.0.1.tar.gz
+    cd horton-2.0.1
+
+For Developers
+--------------
+If you have git installed,
+
+.. code-block:: bash
+
+    git clone https://github.com/theochem/horton.git
+
+You may also want to checkout the HORTON release `2.0.0`. It seems to be more stable on Windows.
+
+.. code-block:: bash
+
+    git checkout 2.0.0
+
+
+Check Ubuntu Version (Just In Case)
+===================================
+On bash,
+
+.. code-block:: bash
+
+  lsb_release -a
+
+It should be `14.04.4 LTS`, but other versions may be available.
+
+
+Dependencies for building, installing and testing HORTON
+========================================================
+
+In order to compile and test HORTON, you need to install relatively recent
+versions of the following programs/libraries:
+
+${dependencies_rst}
+
+
+Installing dependencies with a package manager
+----------------------------------------------
+
+With Ubuntu, most of these dependencies can be installed with a `apt-get`:
+
+* **Ubuntu Linux 16.04**:
+
+  .. code-block:: bash
+
+    sudo apt-get install ${dependencies_ubuntu_16_deb}
+
+  Note that Ubuntu 16.04 does not have up-to-date packages for all dependencies. LibInt2
```

Comment:
`MAX_AM_EIR` stands for "maximum angular momentum support for the electron repulsion integrals". That number depends on how you generate the libint source code.